apps/kyb-app/src/pages/CollectionFlow/CollectionFlow.tsx

```diff
@@ -132,7 +133,13 @@ export const CollectionFlow = withSessionProtected(() => {
   if (getCollectionFlowState(initialContext)?.status === CollectionFlowStatusesEnum.failed)
     return <FailedScreen />;
 
-  return definition && collectionFlowData ? (
+  if (!schema || !collectionFlowData) {
+    console.error('Schema is missing.');
```

Comment:
Edge case, shouldnt happen. Otherwise if some properties won't be accesible user will get to `Something went wrong screen` which is handled by error boundary.